ibis/impala/compiler.py

```diff
@@ -170,7 +170,7 @@ def _not_null(translator, expr):
 }
 
 
-def _cumulative_to_window(translator, expr, window):
+def cumulative_to_window(translator, expr, window):
```

Comment:
@semelianova as this method doesn't need to be public in the API, I think you don't need to rename this function.

Reply:
@xmnlab In this case, I get a warning that I try to access a protected member when using this method from omnisci backend (in operation.py)

Reply:
could you give more information about this warning pls? I tried here and I didn't see anything. is this warning raised by your IDE? as you can see there are a few more calls to `impala_compiler` functions that start with underscore inside the same module.

Reply:
Yes, I see it from IDE